lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2354,6 +2386,12 @@ void StmtEmitter::collectNamesEmitDecls(Block &block) {
       printUnpackedTypePostfix(type, os);
     }
 
+    // Constants carry their assignment directly in the declaration.
+    if (isConstantExpression(decl)) {
+      os << " = ";
+      emitExpression(decl->getResult(0), ops, ForceEmitMultiUse);
```

Comment:
Does merging this into emitDeclarationForTemporary eliminate the need for this?

Reply:
Not directly since this part of the emitter (`StmtEmitter::collectNamesEmitDecls`) does not use `emitDeclarationForTemporary`. We would have to extend `emitDeclarationForTemporary` to also cover the use case here (zero bit type handling, type string padding, interface instantiation, unpacked type postfixes). Not sure if that would make things clearer -- reduces some duplication though.

Reply:
ok, lets leave it as-is for now, thx